feat(issue_templates): Add issue templates for bug, features, and game data (#52)
Prophecy-eip/Backend: .github/ISSUE_TEMPLATE/game_data_error.md

---
name: Game data error
about: Report an error in the game data
title: "[E_DATA] Description"
labels: data, bug
assignees: 

___

> ** Before submitting the issue, please check that someone did not request the same update**

Author: ```suggestion
** Before submitting the issue, please check that someone did not request a similar one**
```

## Game
- [ ] The 9th Age
- [ ] Other:

## Army book
*Book's name*

**Version:**

## Data type
- [ ] Army
- [ ] Unit category
- [ ] Unit
- [ ] Upgrade category
- [ ] Upgrade
- [ ] Special item category
- [ ] Special item
- [ ] Rule
- [ ] Other:

> ## Description

Author: ```suggestion
## Detailed description
```

*Description of the error*

## Expected

> *What you expect**

Author: ```suggestion
*What you expect*
```
